dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/content/util/ContentUtilsTest.java

```diff
@@ -794,4 +797,116 @@ public void testSortPullRelated(final SortTestCase testCase) throws DotDataExcep
         }
     }
 
+    /**
+     * Method to test: {@link ContentUtils#pull(String, int, String, User, String)}
+     * When: there is a content with a publish date in the future and the time machine parameter in null
+     * Should: Not return the content
+     */
+    @Test
+    public void whenTheTimeMachineDateIsNullAndPublishDateInFutureShouldNotReturnAnything() {
+        final String timeMachine = null;
+        final Calendar contentPublishDate = Calendar.getInstance();
+        contentPublishDate.add(Calendar.DATE, 1);
+
+        final ContentType contentType = new ContentTypeDataGen().nextPersisted();
+        final Contentlet contentlet = new ContentletDataGen(contentType.id())
+                .setPolicy(IndexPolicy.FORCE)
+                .setProperty("sysPublishDate", contentPublishDate.getTime())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "sysPublishDate" appears 4 times in this file; the first occurrence is on line 814](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4740221)